### PR TITLE
Feat/brl 33 application setup

### PR DESCRIPTION
**Pull request by vvandijck**
This pull request includes:
- Multiple tools are supported and tools can now be added separately through `.json` files.
- Components belonging to each tool can be added to the `components` folder, just like we're used to.
- The new `Step1`, `Step2` and `Step3` views are quite 'dumb', in the sense that they only handle navigation related things.
- I've added some stub data and component to show how/test how everything works.
- I've refactored the stores and split it up in separate modules, using the same patterns to talk to the store.

To-do:
- [ ] test `brl` tool to make sure it's on par with the current implementation.
- [ ] implement any feedback

### CODE REVIEW

File: src/components/load-button.vue [hunk 1]
```diff
@@ -41,18 +49,23 @@
...
     },
     methods: {
-      ...mapActions({
-        showError: 'notifications/showError',
-      }),
+      ...mapActions('data', [ 'loadProject', 'reset' ]),
+      ...mapActions('mapbox', [ 'getFeature' ]),
+      ...mapActions('selections', [ 'addSelection' ]),
+      onClick() {
+        this.$refs.uploader.click();
```
Comment (sjoerdbeentjes) [inline src/components/load-button.vue:64]: Why is this needed?

File: src/components/some-new-component/some-new-component.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        foo: 'bar',
+      };
+    },
+    computed: {
+      ...mapGetters('app', [ 'viewerCurrentStep' ]),
+    },
+  };
+</script>
+
+<style src="./some-new-component.css" lang="css"></style>
```
Comment (sjoerdbeentjes) [inline src/components/some-new-component/some-new-component.vue:22]: Does this serve as an example component or is it a leftover?

**Review by sjoerdbeentjes — APPROVED** [review]
Looking very good! 💯 added some small comments, but don't let them hold you back merging this PR

File: src/components/some-new-component/some-new-component.vue (hunk 2 above)
Comment (vvandijck) [reply src/components/some-new-component/some-new-component.vue:22]: As an example for us. It will probably already be removed when we start working on the next tickets.

File: src/components/load-button.vue (hunk 1 above)
Comment (vvandijck) [reply src/components/load-button.vue:64]: Instead of clicking on an 'invisible' input field to trigger the upload UI, where the buttons hover/focus effects would not show. We now use a button, have hover/focus styles, and when clicking on it we trigger a click on the 'hidden' input field.
